pandas/tests/extension/test_categorical.py

```diff
@@ -197,7 +197,14 @@ def test_searchsorted(self, data_for_sorting):
 
 
 class TestCasting(base.BaseCastingTests):
-    pass
+    @pytest.mark.parametrize("cls", [Categorical, CategoricalIndex])
+    def test_cast_nan_to_int(self, cls):
```

Comment:
Can you also parametrize this by datetime values like 

```
[pd.Timestamp('2000'), pd.NaT]
```

in addition to the numeric ones you already have?